packages/gestalt/src/__snapshots__/ModalAlert.jsdom.test.js.snap

```diff
@@ -313,7 +313,7 @@ exports[`ModalAlert Desktop ModalAlert renders (warning) 1`] = `
               </div>
             </div>
             <div
-              class="box flexGrow overflowAuto paddingX6 paddingY6 relative"
+              class="box flexGrow overflowAuto paddingX6 paddingY6 relative xsDisplayFlex"
```

Comment:
I'm not able to verify the changes here, Modal alert doesn't seem to be working in devexample mode.

But I noticed this adds a 
```
 display: flex;
 flex-direction: row;
  ```
  
So wondering if anything with multiple children here works as expected, the ordering is different, since the other classes don't seem to add a flex layout explicitly

Reply:
<img width="555" alt="Screenshot by Dropbox Capture" src="https://github.com/pinterest/gestalt/assets/10593890/eb7fade3-7e69-4f62-86f9-e0e9f76af5c4">

Reply:
This Box is following the previous Box wrapping the  footer code to keep the same positioning and stickiness and all.

<img width="668" alt="Screenshot by Dropbox Capture" src="https://github.com/pinterest/gestalt/assets/10593890/2be0cb81-dc4c-4952-9a9b-8fa5c746dbf3">

Without this it would not stay in place.

Reply:
Reviewing my comments... to see if it makes actual sense ha

Reply:
REMOVED DONE